torch/_inductor/fx_passes/post_grad.py

```diff
@@ -704,7 +704,10 @@ def reinplace_inplaceable_ops(graph):
 
             copy_args_to_copy_nodes[(dst, src)] = node
 
-            assert node.args[0].op == "placeholder"
+            # "graph inputs" can come from a get_attr instead of a placeholder,
+            # if we have "unlifted" params/buffers back into state on the GraphModule
+            # (we currently do this in the export flow)
+            assert node.args[0].op == "placeholder" or node.args[0].op == "get_attr"
```

Comment:
cc @Chillee  / @oulgen (fyi, this happens because the AOTInductor workflow "unlifts" buffers, so they're buffers on the GraphModule instead of graph inputs).

There are also some failing tests.. gonna look into them soon